### DISCUSSION TITLE
Forward chosen global shortcuts to chosen windows

### DISCUSSION THREAD

**Discussion by salman-farooq-sh** [YaLTeR/niri#427] [Ideas]
[As in Hyprland](https://wiki.hyprland.org/Configuring/Binds/#global-keybinds).

**Comment by YaLTeR** [comment]
Not sure if there's anything that will require changes in Smithay (we need to be careful with keyboard enter/leave events). I guess we can start with an action that sends a press+release of a specific key to a window matched similar to window rules.

**Reply by salman-farooq-sh** [reply]
Or maybe a window rule property which passes the specified keys/combinations to matched window/s, instead of the active window.

Just different config approach, same thing otherwise.

**Reply by YaLTeR** [reply]
I'll need to see what makes more sense.
